science/dftd3/Portfile

```
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                dftd3
+version             3.2.0
+revision            0
+categories          science physics chemistry
+license             public-domain
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Dispersion Correction For Quantum Chemical Methods
+long_description    Dispersion correction for density functionals, Hartree-Fock \
+                    and semi-empirical quantum chemical methods
+homepage            https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3
+master_sites        ${homepage}
+distfiles           dftd3.tgz
```

This can probably be set using variables:

```suggestion
extract.suffix      .tgz
distfiles           ${name}${extract.suffix}
```

---

nice. I found more places for using ${name}.

---

I just realized that this may pose a problem when MacPorts mirrors the distfile for this port.

Since the distfile's name stays the same across different versions, this might create issues in the mirror.

I think we will want to preemptively add a `dist_subdir` line:

```
dist_subdir         ${name}/${version}
```

This will create a new subdirectory in the mirrors for each version where the consistently named distfile will be stored.

---

The project has been very quiet. So, I don't expect this to become a major issue, but it's always better to be on the safe side. Also this is not that much work.